src/__tests__/testScreen.py

```diff
@@ -196,4 +207,16 @@ def maybeMakeExpectedDir():
             os.makedirs(EXPECTED_DIR)
 
 if __name__ == '__main__':
+    # monkey patch clearLine to clear the input region, not print spaces
+    # this matches the intent of the function and makes sure we don't
+    # generate empty spaces in the output for comparing against the expected
+    # input
+
+    def clearLine(self, y):
+        (minx, miny, maxx, maxy) = self.getChromeBoundaries()
+        for x in range(minx, maxx):
+            self.stdscr.output[(x, y)] = ('', 1)
+
+    screenTestRunner.choose.screenControl.Controller.clearLine = clearLine
```

Comment:
I'm sort of OK with this specific instance of monkey patching, because I'd rather monkey patch `ncurses` itself. But you can't because it's a compiled module, so I had to replace the function in `screenController`.  If you'd like we can move this function into `cursesAPI`, and provide a wrapper around the existing `cursesAPI` when running against real `ncurses`. 

Actually this would all be better if the controller was separate from the view code here, but that's another refactor for another day.